criticality_score/run.py

```diff
@@ -627,10 +683,18 @@ def main():
         default=[],
         help='Additional parameters in form <value>:<weight>:<max_threshold>',
         required=False)
+    parser.add_argument(
+        '--overrides',
```

Comment:
Is there any sense in combining `--overrides` with `--params`?

Reply:
I've actually thought that --params should support specifying a name and the results should display those along with the built-in parameters (they currently aren't), if we did so we could combine the two with a syntax such as:
`--params <name>:<weight>:<max_threshold>[:<value>]` specifying that if the name matches a built-in param it is an override.
On the other hand this would make the command usage more complicated to understand. The separate option has the advantage of being more explicit.

Reply:
Agreed. Happy to leave it separate